app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,59 @@
+class PasswordResetsController < ApplicationController
+  before_action :load_user, :valid_user, :check_expiration,
+                only: [:edit, :update]
+
+  def new; end
+
+  def edit; end
+
+  def update
+    if @user.update user_params
+      @user.update_columns reset_digest: nil
```

Comment:
```suggestion
      @user.update_column :reset_digest, nil
``` cập nhật 1 field cần j dùng `columns` em ơi